### PR TITLE
Upgrade to latest version 0.94j

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@isuruf: This release adds `cddexec` which replaces `cdd_both_reps`. However, this makes this a non-backwards-compatible change. Is there any way we can signal this in the recipe? Or should we add `cdd_both_reps` again until every dependency has upgraded (how do we know?)

**Comment by isuruf** [comment]
You mean that cddlib breaks ABI/API on patch level?

2 options.
1. Change run_exports in 1!0.94h so that it pins exactly. Rebuild gfan, singular, sage and mark previous builds as broken
2. Change the version to 2!0.94j

Btw, can you rebase on top of master?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,14 @@
 {% set name = "cddlib" %}
-{% set version = "0.94i" %}
-{% set sha256 = "c60dac8697357740c593f8f255d49ac8a5069623561d68720fd9089367c90f4a" %}
+{% set version = "2!0.94j" %}
+{% set sha256 = "27d7fcac2710755a01ef5381010140fc57c95f959c3c5705c58539d8c4d17bfb" %}
 
 package:
   name: {{ name }}
   version: 1!{{ version }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Need to remove `1!` here

**Comment by saraedum** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (saraedum) [reply recipe/meta.yaml]: Sure, actually from the version variable because that one is also used to build the download URL.

**Comment by saraedum** [comment]
@isuruf: Does this look ok to you now?

**Comment by isuruf** [comment]
I sitll don't understand why you need to increase the epoch.

**Comment by saraedum** [comment]
I thought you had proposed that in https://github.com/conda-forge/cddlib-feedstock/pull/6#issuecomment-399157026 essentially. We won't build cdd_both_reps anymore(, and cddlib is moving to semantic versioning.)

**Comment by saraedum** [comment]
So, as only conda-forge/sage-feedstock depends on `cdd_both_reps`, it seems to be safe to keep the epoch as is once we merged https://github.com/conda-forge/sage-feedstock/pull/19.

**Pull request merged**